test/cpp/end2end/crl_provider_test.cc

```diff
@@ -220,6 +221,48 @@ TEST_F(CrlProviderTest, CrlProviderRevokedServer) {
   DoRpc(server_addr_, options, false);
 }
 
+TEST_F(CrlProviderTest, CrlProviderValidReloader) {
```

Comment:
To make sure I understand correctly, the difference between this test and the `CrlProviderValid` test is that this test reloads from a directory whereas the other test is explicitly given the CRL at construction time. Is that correct?

Could you edit the test names to help clarify the difference? And/or maybe add a comment to explain this?

Reply:
This is the same e2e test but with the directory reloader vs. the static provider

Reply:
Thanks for clarifying!